### PR TITLE
Implemented CRUD (but testing still needed)

### PR DESCRIPTION
**Pull request by Kraig5**

### CODE REVIEW

File: database/index.js [hunk 1]
```diff
@@ -28,7 +45,118 @@ const getPhotos = (expId, callback) => {
...
+    if (error) {
+      callback(error);
+    } else {
+      getExperiencePhotos(expId, (err, response) => callback(null, response));
+    }
+  });
+};
+
+const putExperiencePhotos = async (expIdDangerous, array, callback) => {
+  // This function is for updating an expId. There are two kinds of updates allowed.
```
Comment (priscilagonzalez) [inline database/index.js]: The comments helped me to understand this function. I

File: server/index.js [hunk 2]
```diff
@@ -22,7 +22,7 @@ app.get('/:id', (req, res) => {
 
 app.get('/photos/:id', (req, res) => {
   const requestExpId = parseInt(req.params.id, 10);
-  db.getPhotos(requestExpId, (err, exData) => {
+  db.getExperiencePhotos(requestExpId, (err, exData) => {
     if (err) {
       res.status(400).send(err);
     } else {
```
Comment (priscilagonzalez) [inline server/index.js]: What happens if there is no error?

Comment (priscilagonzalez) [inline server/index.js]: What happens if there is no error?

Comment (Kraig5) [reply server/index.js]: github seems to have hidden line 29 which says "res.status(200).send(exData)", so I think it's OK

**Pull request merged**